src/main/resources/assets/plethora/lang/de_DE.lang

```diff
@@ -0,0 +1,73 @@
+item.plethora.module.module_laser.desc=Feuert verdammte Laser.
+
+death.attack.laser=%1$s wurde von %2$s aufgelöst
+death.attack.laser.item=%1$s wurde von %2$s mit %3$s aufgelöst
+entity.Plethora.laser.name=Laser
+
+item.plethora.module.module_scanner.name=Blockscanner
+item.plethora.module.module_scanner.desc=Scannt Blöcke in der Umgebung
+
+item.plethora.module.module_sensor.name=Entitysensor
+item.plethora.module.module_sensor.desc=Erkennt naheliegende Entities und sammelt dessen Daten
+
+item.plethora.module.module_kinetic.name=Kinetischer Verstärker
+item.plethora.module.module_kinetic.desc=Ermöglicht Kontrolle über deinen Körper mit Hilfe einer neuralen Schnittstelle
+
+item.plethora.module.module_unknown.name=Unbekanntes Modul
+item.plethora.module.module_unknown.desc=Was hast du jetzt schon wieder angestellt?
+
+tile.plethora.manipulator.mark_1.name=Manipulator Stufe I
+tile.plethora.manipulator.mark_2.name=Manipulator Stufe II
+tile.plethora.manipulator.mark_1.desc=Gewährt einem Computer Zugriff auf ein Modul
+tile.plethora.manipulator.mark_2.desc=Gewährt einem Computer Zugriff auf mehrere Module
+
+# Neural interface/connector
+item.plethora.neuralConnector.name=Neuraler Verbinder
+item.plethora.neuralConnector.desc=Wird zur Verbindung mit neuralen Schnittstellen verwendet
+
+item.plethora.neuralInterface.name=Neurale Schnittstelle
+item.plethora.neuralInterface.desc=Wird als Helm getragen und mit dem Neuralen Verbinder verbunden.
+
+gui.plethora.neuralInterface.peripherals=Peripherals anzeigen
+gui.plethora.neuralInterface.modules=Module anzeigen
+
+# Creative tab
+itemGroup.plethora=Plethora
+
+# Core config
+gui.config.plethora.costsystem=Preissystem
+gui.config.plethora.costsystem.initial=Anfangslevel
+gui.config.plethora.costsystem.regen=Regeneration
+gui.config.plethora.costsystem.limit=Grenze
+gui.config.plethora.blacklist=Sperrliste
+gui.config.plethora.blacklist.blacklistMethods=Gesperrte Methoden
+gui.config.plethora.blacklist.blacklistMeta=Gesperrte meta provider
+gui.config.plethora.blacklist.blacklistTileEntities=Gesperrte TileEntities
+gui.config.plethora.baseCosts=Grundpreis für Methoden
+gui.config.plethora.testing=Prüfen
+
+# Gameplay config
+gui.config.plethora.laser=Verdammter Laserstrahl
+gui.config.plethora.laser.laserMinimum=Laser Minimum
+gui.config.plethora.laser.laserMaximum=Laser Maximum
+gui.config.plethora.laser.laserCost=Laserpreis
+
+gui.config.plethora.kinetic=Kinetischer Verstärker
+gui.config.plethora.kinetic.launchMax=Maximaler Schleuderfaktor
+gui.config.plethora.kinetic.launchCost=Maximaler Schleuderpreis
+gui.config.plethora.kinetic.walkRange=Maximale Laufreichweite
+gui.config.plethora.kinetic.walkSpeed=Maximale Laufgeschwindigkeit
+gui.config.plethora.kinetic.walkCost=Laufkosten
+gui.config.plethora.kinetic.teleportRange=Maximale Teleportationsreichweite
+gui.config.plethora.kinetic.teleportCost=Teleportationspreis
+
+gui.config.plethora.scanner=Entityscanner
```

Comment:
Both plethora.scanner and plethora.sensor are "Entity Sensor" in en_US.lang, that also seems wrong.